src/app.ts

```diff
@@ -4,10 +4,10 @@ import * as helmet from "koa-helmet";
 import { ApolloServer } from "apollo-server-koa";
 
 import { schema } from "./schema";
-
+import { createContext } from "context";
```

Comment:
현재 파일과 같은 위치에 있어서 다음처럼 바꾸는게 좋을 것 같습니다
```suggestion
import { createContext } from "./context";
```